### PR TITLE
Home work1

### PR DESCRIPTION
**Pull request by Illia1927**

### CODE REVIEW

File: src/mate/academy/homework7/directory/annotation/InjectorHuman.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public static void injectDependency() throws IllegalAccessException {
+
+        Class consoleHumanHandlerClass = ConsoleHandlerHuman.class;
+        Class fileHumaDaoClass = FileHumanDao.class;
+        Class inMemoryHumanDaoClass = InMemoryHumanDao.class;
+
+        Field[] fields = consoleHumanHandlerClass.getDeclaredFields();
+
+        for (Field field : fields) {
+            if(field.isAnnotationPresent(Inject.class)){
```
Comment (Siril-O) [inline src/mate/academy/homework7/directory/annotation/InjectorHuman.java]: format whitespaces

File: src/mate/academy/homework7/directory/annotation/InjectorHuman.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import mate.academy.homework7.directory.dao.InMemoryHumanDao;
+import mate.academy.homework7.directory.factory.HumanDaoFactory;
+import mate.academy.homework7.directory.handler.ConsoleHandlerHuman;
+
+import java.lang.reflect.Field;
+
+public class InjectorHuman {
+    public static void injectDependency() throws IllegalAccessException {
+
+        Class consoleHumanHandlerClass = ConsoleHandlerHuman.class;
```
Comment (Siril-O) [inline src/mate/academy/homework7/directory/annotation/InjectorHuman.java]: create field with list of classes and make this method to be generic, not bounded to 3 classes, it should work for any class you pass

File: src/mate/academy/homework7/directory/dao/FileHumanDao.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    @Override
+    public Human getHuman() {
+        try {
+            ObjectInputStream objectInputStream = new ObjectInputStream(
+                    new FileInputStream("HumanDirectory.txt"));
+            return (Human) objectInputStream.readObject();
+        } catch (Exception e) {
+            System.err.println("Human not found in system");
+        }
+        return null;
```
Comment (Siril-O) [inline src/mate/academy/homework7/directory/dao/FileHumanDao.java]: do not return null

**Review by Siril-O — CHANGES_REQUESTED** [review]

File: src/mate/academy/homework7/directory/annotation/InjectorHuman.java (hunk 2 above)
Comment (Illia1927) [reply src/mate/academy/homework7/directory/annotation/InjectorHuman.java]: @Siril-O Why you don`t check my home work again? I have 4 red HW.... Please, check it!

File: application.properties [hunk 4]
```diff
@@ -0,0 +1,2 @@
+db.name=file
```
Comment (Siril-O) [inline application.properties:1]: Why you have 7 homeworks in pull request to 1 homework ?

**Review by Siril-O — CHANGES_REQUESTED** [review]

File: application.properties (hunk 4 above)
Comment (Illia1927) [reply application.properties:1]: @Siril-O I did it by chance, when I close the pool request, everything will be fine

**Pull request merged**